packages/flutter_tools/lib/src/dart/analysis.dart

```diff
@@ -46,6 +46,7 @@ class AnalysisDriver {
   DriverOptions options;
 
   String get sdkDir {
+    // ignore: deprecated_member_use
```

Comment:
I don't know if anyone is using those features - for now I'd just like to get the analyzer running cleanly again.

Can you or @pq remove the getSdkDir() call if it's no longer necessary?